scripts/rnnlm/get_unigram_probs.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
+
+import os
+import argparse
+import sys
+
+parser = argparse.ArgumentParser(description="This script gets the unigram probabilities of words.",
+                                 epilog="E.g. " + sys.argv[0] + " --vocab-file=data/rnnlm/vocab/words.txt "
+                                        "--data-weights-file=exp/rnnlm/data_weights.txt data/rnnlm/data "
+                                        "> exp/rnnlm/unigram_probs.txt",
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+parser.add_argument("--vocab-file", type=str, default='', required=True,
+                    help="Specify the vocab file.")
+parser.add_argument("--data-weights-file", type=str, default='', required=True,
+                    help="Specify the data-weights file.")
+parser.add_argument("--smooth-unigram-counts", type=float, default=1.0,
+                    help="Specify the constant for smoothing. We will add "
+                         "(smooth_unigram_counts * num_words_with_non_zero_counts / vocab_size) "
```

Comment:
I think we should do add-lambda smoothing (lambda here is the smooth-unigram-counts) on unigram counts instead of scaling them right?